changelogs/unreleased/schema-new-replicaset-uuid-key.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/core
+
+* **[Breaking change]** The key `box.space._schema['cluster']` is renamed to
```

Comment:
*"This is our internal key"* - it is documented, like I said in the docbot request.
https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_space/_schema/. Hence people could be using it, for example, to fetch replicaset UUID via a remote connection directly from `_schema` space. Thus it is a breaking change.

Reply:
The `_schema` space is a system space so it shouldn't be accessed directly by users. Our connectors don't use the "cluster" key either AFAIK. Besides, the change will only occur if the user manually calls `box.schema.upgrade()` and there's a way to revert (downgrade) so I think it should be safe. Up to @Totktonada.

Reply:
I don't know actual usages. It would be a gray zone, but it is actually documented, so no. That's technically breaking and I think it is fair to mark it as such. We can leave some words that we don't expect any actual code breakage, because X and Y.

Reply:
I kept the "breaking" label and explicitly mentioned that it is not supposed to really break anything, but marked like that because of the documentation and a little-tiny chance that somebody could rely on that.